resources/public/api/conf/2.0/application.raml

```diff
@@ -391,12 +391,10 @@ traits:
                             "href": "/customs/transits/movements/departures/62f4ebbbf581d4aa"
                           }
                         },
-                        "departure":{
-                          "id": "/customs/transits/movements/departures/62f4ebbbf581d4aa"
-                        },
-                        "id": "/customs/transits/movements/departures/62f4ebbbf581d4aa/messages/62f4ebbb765ba8c2",
+                        "id": "62f4ebbb765ba8c2",
+                        "departureId": "62f4ebbbf581d4aa",
                         "received": "2022-08-11T11:44:59.83705",
-                        "messageType": "IE015",
+                        "type": "IE015",
```

Comment:
No, this should be `messageType`, as per https://github.com/hmrc/common-transit-convention-traders/blob/3991baefbe9aa83be97973a9cdc6c58335a6e74e/app/models/response/HateoasDepartureResponseMessage.scala#L40

Reply:
Liandra has modified it and it is now using HateoasMovementMessageResponse and there it is using  "type"                            -> messageSummary.messageType

Reply:
Yup, you're right, my mistake, it is `type`. Carry on!